src/govuk/components/notification-banner/_index.scss

```diff
@@ -0,0 +1,87 @@
+@include govuk-exports("govuk/component/notification-banner") {
+  /* stylelint-disable */
+
+  .govuk-notification-banner {
+    @include govuk-font($size: 19);
+    border: $govuk-border-width solid $govuk-brand-colour;
+
+    &:focus {
+      outline: 4px solid $govuk-focus-colour;
+    }
+  }
+
+  .govuk-notification-banner--success {
+    border-color: govuk-colour("green");
+  }
+
+  .govuk-notification-banner--error {
+    border-color: $govuk-error-colour;
+  }
+
+  .govuk-notification-banner__header {
```

Comment:
It'd be interesting to take a look at this when colours are overridden (in the browser, or in Windows high contrast mode) as the background gets removed so the header and body 'merge':

![Screenshot 2020-10-13 at 09 51 10](https://user-images.githubusercontent.com/121939/95838410-a5269c00-0d39-11eb-8952-c99e23a45100.png)

For example, adding a bottom border to the header (which could be the same colour as the background, or transparent):

![Screenshot 2020-10-13 at 09 51 00](https://user-images.githubusercontent.com/121939/95838405-a48e0580-0d39-11eb-95df-034b77f7fed8.png)

/cc @christopherthomasdesign

Reply:
Think it deffo looks nicer with the border – looks more structured and the spacing of 'important' looks a bit too close to the top without it. Happy to go with that if it's not going to show in other contexts. Transparent sounds better, are there any downsides to that?

Reply:
@christopherthomasdesign Let me have play around with both options and see 👍

Reply:
Added a transparent border to the header [here](https://github.com/alphagov/govuk-frontend/pull/1982/files#diff-d29c1ca21cb393b72322bb9bc3092fee876d24637c3611404cfde71a8af7fa07R26)

Example:

<img width="961" alt="Screenshot 2020-10-13 at 13 02 04" src="https://user-images.githubusercontent.com/29889908/95986670-4850e180-0e1e-11eb-917a-4d3d7ea9f20f.png">